Gopkg.toml

```diff
@@ -246,6 +246,10 @@
   name = "github.com/vishvananda/netlink"
   revision = "b2de5d10e38ecce8607e6b438b6d174f389a004e"
 
+[[constraint]]
+  name = "github.com/wavefronthq/wavefront-sdk-go"
+  version = "v0.9.0"
```

Comment:
There should be some changes to Gopkg.lock as well, which is why the tests aren't working, just need to run `dep ensure` once.